docs/moment/00-use-it/08-typescript.md

```diff
@@ -12,29 +12,31 @@ npm install moment
 
 Import and use in your Typescript file
 <!-- skip-example -->
+
 ```javascript
 import * as moment from 'moment';
 
-
 let now = moment().format('LLLL');
 ```
 
-**Note:** If you have trouble importing moment, try adding ```"allowSyntheticDefaultImports": true``` in ```compilerOptions``` in your ```tsconfig.json``` file and then use the syntax
+**Note:** If you have trouble importing moment, try adding ```"moduleResolution": "node"``` in ```compilerOptions``` in your ```tsconfig.json``` file and then use the either of the below syntax
```

Comment:
Thanks for submitting this request.

My guess is that @andy1547 's problem is a new Typescript thing. In older versions of Typescript, the old advice may still apply.

I would prefer that you add another "**Note:**" with additional advice, as opposed to modifying the existing advice.

Reply:
I had tested the new approach as well as the existing approach with the `"typescript": "2.0.0"` and also with its higher versions`(not checked below 2.0.0)`. As per the existing advice, I was getting a compilation error. Not sure up-to which version of typescript existing advice is valid. 

So, @marwahaha as the new approach valid for the `typescript 2.0.0`. Do we still need to have the existing approach in the documentation?

Reply:
I'd prefer to have it; it's possible there are Typescript 1.X users still on Moment 2.x and we haven't explicitly dropped support for them.